crates/wiggle/src/lib.rs

```diff
@@ -627,7 +722,7 @@ impl<'a> GuestPtr<'a, str> {
         let ptr = unsafe { slice::from_raw_parts_mut(ptr, self.pointer.1 as usize) };
```

Comment:
This `unsafe` could use a coment similar to above; `SAFETY: iff there are no overlapping borrows it is ok to construct a &mut str`.